frontend/src/InteractiveImport/Interactive/InteractiveImportModalContent.tsx

```diff
@@ -131,7 +131,7 @@ const COLUMNS = [
       name: icons.INTERACTIVE,
       title: 'Custom Format',
     }),
-    isSortable: true,
```

Comment:
This should be sortable, if it's not working correctly then it might need a `sortPredicate` in `interactiveImportActions`

Reply:
Adding a sortPredicate fixed the problem I was noticing. Thanks for pointing that out, didn't know how sorting is implemented